modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -986,6 +988,17 @@ def set_derived_appointment_date_fields(appointment)
         appointment[:future] = future?(appointment)
       end
 
+      def log_telehealth_issue(appointment)
+        context = {
+          kind: appointment[:kind],
+          vvsVistaVideoAppt: appointment.dig(:extension, :vvsVistaVideoAppt),
+          modality: appointment[:modality],
+          telehealthUrl: appointment.dig(:telehealth, :url),
+          start: appointment.dig(:start)
+        }
+        Rails.logger.warn("VAOS video telehealth issue", context.to_json)
```

Comment:
I know DD logs filtering is pretty powerful but we might want to limit this to only error cases, i.e. when displayLink or telehealthUrl is missing. It could makes it easier to find the relevant logs and I don't think the non-error cases are interesting to inspect.

Reply:
Sounds good. I imagine the url missing would be the most general cause that could lead us to narrow with the data.